data/data-pipeline/data_pipeline/etl/sources/tribal_overlap/etl.py

```diff
@@ -246,6 +249,12 @@ def transform(self) -> None:
             field_names.COUNT_OF_TRIBAL_AREAS_IN_TRACT_CONUS
         ] = None
 
+        merged_output_df[field_names.IS_TRIBAL_DAC] = np.where(
```

Comment:
I think it might be a bit cleaner to do this with a fillna potentially?

````
merged_output_df[field_names.IS_TRIBAL_DAC] = merged_output_df[field_names.PERCENT_OF_TRIBAL_AREA_IN_TRACT]
            > self.TRIBAL_OVERLAP_CUTOFF
```